spec/features/request_ride_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+feature "Request ride" do
+  let(:user) { create(:user) }
+  let!(:ride) { create(:ride) }
+  before(:each) { sign_in user }
+
+  context "when ride available" do
+    before(:each) do
```

Comment:
You might not want to run this for every test. If this path works once, you will just be increasing the test time.